dub.sdl

```diff
@@ -216,6 +218,8 @@ subPackage {
 	libs "sokol"
 	lflags "-Lzig-out/lib" platform="posix"
 	lflags "/LIBPATH:zig-out/lib" platform="windows"
+	dflags "-P-I$HOME/.cache/zig/p/1220190c1c465474d437a8f7955c8b759e6e4ef32e9a61e9f0b5dbe74ebcd84622b3/src" platform="posix"
```

Comment:
Although it's not ideal, it works. Instead of `cImportPaths`/`cSourcePaths` which are already available in the dub.

#### References
- https://dlang.org/changelog/2.104.0.html#collect_c_source_and_headers_for_ImportC
- https://github.com/dlang/dub/issues/2698